### ISSUE TITLE
Use ISO 8601 for formatting dates in API calls

### ISSUE THREAD

**Issue by pacificgilly1992** [ExpDev07/coronavirus-tracker-api#19]
I’ve noticed that you use the American date time format MM/DD/YY for the dates returned in the API calls despite you using iso formatted dates in parts of your source code. Could we make this API a lot more robust and stick to a sensible date time format such as ISO 8601, to avoid guess work.

**Comment by ExpDev07** [comment]
Apologies for that. The JSU data repository where we retrieve the data provides the "MM/DD/YY" date format, which was initially used. Changing this now will cause a lot of people's apps to break, which would not be good. The ``last_updated`` field was added today, and I went with the ISO 8601 format as it was better suited for that specific field.

If, in the future, when we decide to make major changes to how the data is being returned, we will go with the ISO 8601 format for the locations too. This will of course be under a ``/v2/`` prefix and the old endpoints will stay for legacy reasons.
